editor/src/components/navigator/layout-element-icons.ts

```diff
@@ -25,10 +26,7 @@ export function useLayoutOrElementIcon(path: ElementPath): LayoutIconResult {
     },
     'useLayoutOrElementIcon',
     (oldResult: LayoutIconResult, newResult: LayoutIconResult) => {
-      return (
-        oldResult.hasWidthOrHeight === newResult.hasWidthOrHeight ||
-        shallowEqual(oldResult.iconProps, newResult.iconProps)
-      )
+      return shallowEqual(oldResult.iconProps, newResult.iconProps)
```

Comment:
I think a isPositionAbsolute comparison is missing from here

Reply:
oh, of course. Fixed.